backend/FwLite/LcmCrdt/Changes/ExampleSentences/UpdateTranslationChange.cs

```diff
@@ -19,7 +19,7 @@ public UpdateTranslationChange(Guid entityId, Guid translationId, JsonPatchDocum
 
     public override ValueTask ApplyChange(ExampleSentence entity, IChangeContext context)
     {
-        var translation = entity.Translations.FirstOrDefault(t => t.Id == TranslationId);
+        var translation = entity.Translations.SingleOrDefault(t => t.Id == TranslationId);
```

Comment:
even though there should never be more than 1 with a given Id, this would crash the app it if was false, I'd rather use First which might do weird stuff in that case, or just apply it to all that match the id?